src/components/Actions/actions.js

```diff
@@ -0,0 +1,69 @@
+import { CALL_API } from 'redux-api-middleware';
+import * as actionTypes from './actionTypes';
+
+export const fetchCar = () => ({
+  [CALL_API]: {
+    types: [
+      actionTypes.FETCH_CAR,
+      actionTypes.FETCH_CAR_SUCCESS,
+      actionTypes.FETCH_CAR_FAILURE,
+    ],
+    endpoint: 'http://apis.is/car?number=fma08',
+    method: 'GET',
+  },
+});
+
+export const fetchAuthenticationData = () => {
+  console.log('fetch auth data');
+  return {
+    [CALL_API]: {
+      types: [
+        actionTypes.FETCH_AUTENTICATION,
+        {
+          type: actionTypes.FETCH_AUTENTICATION_SUCCESS,
+          payload: (action, state, res) => {
+            return {
+              token: res.headers.get('authorization'),
+            };
+          },
+        },
+        actionTypes.FETCH_AUTENTICATION_FAILURE,
+      ],
+      endpoint: 'https://test-api.sling.is/v1/account/login',
```

Comment:
`https://test-api.sling.is/v1/` should be in .env file since if you had to test e.g. `https://test-api.sling.is/v2/` you would only need to replace it in a single location.

Your endpoint would look something like 
```javascript
endpoint: `${process.env.REACT_APP_API}${'account/login'}`,
```